sdk/storage/Azure.Storage.Blobs/README.md

```diff
@@ -1,6 +1,6 @@
 # Azure Storage Blobs client library for .NET
 
-> Server Version: 2019-12-12, 2019-07-07, and 2019-02-02
+> Server Version: 2020-02-10, 2019-12-12, 2019-07-07, and 2020-02-02
```

Comment:
Change the last version back to 2019-02-02

Reply:
fixed.

Reply:
fixed.